pkg/debuginfo/store.go

```diff
@@ -71,12 +73,14 @@ type Store struct {
 }
 
 // NewStore returns a new debug info store
-func NewStore(logger log.Logger, symbolizer *symbol.Symbolizer, config *Config) (*Store, error) {
+func NewStore(logger log.Logger, symbolizer *symbol.Symbolizer, config *Config, debuginfodUrl string) (*Store, error) {
 	cfg, err := yaml.Marshal(config.Bucket)
 	if err != nil {
 		return nil, fmt.Errorf("marshal content of object storage configuration: %w", err)
 	}
 
+	debuginfodClientCache := NewObjectStorageDebugInfodClientCache(NewHttpDebugInfoClient(debuginfodUrl))
```

Comment:
I think this client should be instantiated by the caller of this function instead. The store should not need to care that what it uses is a cached client, it should just accept a parameter that implements the `DebugInfodClient` interface.